src/components/primitives/Input/index.tsx

```diff
@@ -157,6 +157,12 @@ const Input = (
           editable={isDisabled || isReadOnly ? false : true}
           // borderRadius={50} //Remove variant props from StyledInput
           borderWidth={undefined}
+          {...(Platform.OS === 'web'
+            ? {
+                disabled: isDisabled,
```

Comment:
Would this only be for web?

Reply:
Adding disable on make the component unfit for focus. That's why adding disabled only for the web.